Gemfile.lock

```diff
@@ -244,6 +244,9 @@ GEM
       actionpack (>= 3.0)
       activesupport (>= 3.0)
       sprockets (>= 2.8, < 4.0)
+    sqlite3 (1.3.10)
```

Comment:
It comes with the gem...

On Sunday, January 11, 2015, Joel Low <notifications@github.com> wrote:

> In Gemfile.lock
> <https://github.com/Coursemology/coursemology2/pull/41#discussion-diff-22765354>
> :
>
> > @@ -244,6 +244,9 @@ GEM
> >        actionpack (>= 3.0)
> >        activesupport (>= 3.0)
> >        sprockets (>= 2.8, < 4.0)
> > +    sqlite3 (1.3.10)
>
> ...why sqlite3?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Coursemology/coursemology2/pull/41/files#r22765354>.
>